test/jdk/java/lang/constant/ClassDescTest.java

```diff
@@ -133,6 +133,7 @@ public void testPrimitiveClassDesc() throws ReflectiveOperationException {
     public void testSimpleClassDesc() throws ReflectiveOperationException {
 
         List<ClassDesc> stringClassDescs = Arrays.asList(ClassDesc.ofDescriptor("Ljava/lang/String;"),
+                                                        ClassDesc.ofInternalName("java/lang/String"),                                                        ClassDesc.of("java.lang", "String"),
```

Comment:
Looks like an accidental duplicate of `ClassDesc.of("java.lang", "String")`?
```suggestion
                                                        ClassDesc.ofInternalName("java/lang/String"),
```

Reply:
Thanks for pointing it out, I had to watch my screen quite a long time to see it is hidden behind the corner :)